pkg/workloads/onnx_serve/api.py

```diff
@@ -150,8 +154,8 @@ def predict(app_name, api_name):
             util.log_indent("Raw sample:", indent=4)
             util.log_pretty(sample, indent=6)
 
-            if request_handler is not None and util.has_function(request_handler, "preinference"):
-                sample = request_handler.preinference(sample, input_metadata)
+            if request_handler is not None and util.has_function(request_handler, "pre_inference"):
+                sample = request_handler.pre_inference(sample, input_metadata)
```

Comment:
Note: log the result of the pre processing function (if it's defined) (also in `tf/api.py`)